crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -37,6 +38,78 @@ impl DynamicScene {
         Self::from_world(&scene.world, type_registry)
     }
 
+    /// Create a Dynamic Scene with specific entities.
+    ///
+    /// The generic parameter is used as a `Query` filter.
+    ///
+    /// The created scene will include only the entities that match the query
+    /// filter provided. All components that impl `Reflect` will be included.
+    ///
+    /// # Example
+    ///
+    /// Here, the function is creating a scene with all the componentA's in the world, but none of the componentB's.
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    /// use bevy_scene::DynamicScene;
+    ///
+    /// #[derive(Component)]
+    /// struct ComponentA;
+    ///
+    /// #[derive(Component)]
+    /// struct ComponentB;
+    ///
+    /// TODO how to add type registry to the world
```

Comment:
This currently breaks because the world does not have a type registry. So I will need to find some way of adding to it.

Reply:
You should just be able to use init_resource on the World :)

Reply:
ah thank you!